frontend/src/pages/Register/Register.module.css

```diff
@@ -1,12 +1,166 @@
-.registerTitle {
-  font-weight: 500;
-  font-size: 36px;
-  line-height: 42px;
+.page {
+  height: 1024px;
+  background-color: #eff0f0;
+}
+
+.page__window {
+  max-width: 910px;
+  height: 840px;
+  border-radius: 16px;
+  background-color: #ffffff;
+}
+
+.page__wrapper {
+  padding: 56px 234px 104px;
+}
+
+.page__title {
+  font-family: Roboto;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38.4px;
+  letter-spacing: -0.01em;
   text-align: center;
+  margin-bottom: 24px;
+  color: var(--fg-Primary, #081821);
+}
+
+.EmailInput {
+  margin-top: 16px;
+}
+
+.firstNameLabel,
+.LastNameLabel {
+  font-family: Roboto;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 15.36px;
+  letter-spacing: 2%;
+  color: #081821;
+}
+.EmailLabel,
```

Comment:
Why is PascalCase used for these classnames? Is it occasionally or on purpose? If occasionally, let's follow the common pattern and use camelCase.